admin-dev/themes/default/template/controllers/search/helpers/view/view.tpl

```diff
@@ -151,16 +151,24 @@ $(function() {
 
 {/if}
 <div class="row">
-    <div class="col-lg-6">
+    <div class="col-lg-{if $alternativeSearchPanels|@count == 0}6{else}4{/if}">
         <div class="panel">
             <h3>{l s='Search doc.prestashop.com' d='Admin.Navigation.Search'}</h3>
```

Comment:
Right now we have 2 hardcoded search panels (one for PS documentation and one for google... FR 😅 ) then a foreach that render all extra panels

What about we make everything a panel? We remove the 2 hardcoded search panels (delete the HTML markup) and keep only the foreach, then in the Controller we hardcode 2 SearchPanel and use hook to get extra panels

Also maybe we can do a little cleaning and use www.google.com instead of www.google.fr 😆